rcCollectionroutes.js

```diff
@@ -0,0 +1,69 @@
+var express = require('express');
+
+var bodyParser = require('body-parser');
+
+var urlEncoded = bodyParser.urlencoded({ extended: false });
+
+var rcCollection = require('../models/rcCollectionmodel');
+
+var router = express.Router();
+
+app.use(express.static('public'));
```

Comment:
ok corrected

From: jamdon 
Sent: Friday, June 17, 2016 1:23 AM
To: JSCRIPT300-Spring2016/Final_Project 
Cc: seancava ; Author 
Subject: Re: [JSCRIPT300-Spring2016/Final_Project] Final Project Proposal (#10)

In rcCollectionroutes.js:

> @@ -0,0 +1,69 @@
> +var express = require('express');
> +
> +var bodyParser = require('body-parser');
> +
> +var urlEncoded = bodyParser.urlencoded({ extended: false });
> +
> +var rcCollection = require('../models/rcCollectionmodel');
> +
> +var router = express.Router();
> +
> +app.use(express.static('public'));
App isn't defined in this file, so this is a reference error that will prevent your server from running. You can use router instead of app to serve static resources, but I would recommend putting it, meaning your use of the express.static middleware, in your main server file, not in an express router.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.